docs/components/AppHeader.vue

```diff
@@ -7,11 +7,9 @@ const { header } = useAppConfig()
 </script>
 
 <template>
-  <UHeader :ui="{ logo: 'items-center' }">
+  <UHeader>
     <template #logo>
-      <TheLogo class="h-12" />
-      <span>Nuxt</span>
-      <TailwindText class="h-3.5" />
```

Comment:
this keeps the "tailwindcss" text aside from Nuxt text which is appropriate I think

Reply:
`Nuxt` is already in the logo

Reply:
Yes, but the 'Tailwind' brand isn't appropriate I believe - lets continue in https://github.com/nuxt-modules/tailwindcss/pull/873#discussion_r1673711472